components/tikv_util/src/memory.rs

```diff
@@ -75,16 +75,26 @@ impl HeapSize for RaftCmdRequest {
     }
 }
 
+#[derive(Debug)]
+pub struct MemoryQuotaExceeded;
```

Comment:
Are we going to add more information about the failed allocation?

Reply:
No, it's just an error type. I think caller can get all necessary information directly from the current memory quota's APIs. Though it's okay to add more information here if you want.